client/src/components/resources/configs/protocol.js

```diff
@@ -0,0 +1,28 @@
+export const searchResult = ['description_or_abstract']
+
+export const details = [
+  {
+    basic_information: [
+      'protocol_name',
+      'description_or_abstract',
+      'additional_info'
```

Comment:
Is abstract missing here?

Reply:
it is supposed to show description 
but if that isnt defined it will show abstract 
and if that isnt defined it will say description is not specified or empty or something like that